src/Docker.PowerShell/Cmdlets/EnterContainer.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Management.Automation;
+using System.Threading.Tasks;
+using Docker.DotNet.Models;
+using Docker.PowerShell.Support;
+
+namespace Docker.PowerShell.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Enter, "Container",
+            DefaultParameterSetName = CommonParameterSetNames.Default)]
+    public class EnterContainer : SingleContainerOperationCmdlet
+    {
+        protected override async Task ProcessRecordAsync()
+        {
+            var id = Id ?? Container.ID;
+
+            var inspect = await DkrClient.Containers.InspectContainerAsync(id);
+            if (!inspect.State.Running || inspect.State.Paused)
```

Comment:
No, actually. You can attach before you start so that you can guarantee that you see 100% of the data. This is what `docker run` does, for example.

I copied this model from `docker attach`.